### ISSUE TITLE
ADO address may be incorrect

### ISSUE THREAD

**Issue by vikrantvm** [kriswiner/LSM6DSO#2]
https://github.com/kriswiner/LSM6DSO/blob/c6b4518863feffcc598dc71c740887f7a9f1acbe/EmbeddedFunctions/LSM6DSO_SlaveLIS2MDL_SlaveLPS22HB_Dragonfly/LSM6DSO.h#L250

I think 0x6b is the address when SDO/SA0 is high/1 (Table 10 of datasheet)

**Comment by kriswiner** [comment]
Yes, the comment is wrong, but in practice I simply change the address as needed.
